### PR TITLE
Publish SCA Agent to Github releases

### PR DESCRIPTION
**Pull request by dbolharovcmx**

### CODE REVIEW

**Comment by dbolharovcmx** [comment]
[In Jenkins, publish SCA Agent to GitHub releases (via checkbox)](https://trello.com/c/7RhrFbwK/1058-in-jenkins-publish-sca-agent-to-github-releases-via-checkbox)

File: Jenkinsfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ library "pipelineUtils"
 def scaAgentZip
 
 pipeline {
+    parameters {
+        booleanParam(name: 'Release_New_Version', defaultValue: false, description: 'If "true", New release will occur in GitHub')
```
Comment (maor-rosenfeld) [inline Jenkinsfile:7]:
```suggestion
        booleanParam(name: 'releaseNewVersion', defaultValue: false, description: 'If "true", New release will occur in GitHub')
```

File: Jenkinsfile [hunk 2]
```diff
@@ -38,12 +50,18 @@ pipeline {
...
+            }
             steps {
-                script{
-                    scaAgentZip = "sca-agent.${VERSION}.zip"
-                    sh(label: "Create bundle", script: "sh dev/bundle.sh ${scaAgentZip}")
-                    archiveArtifacts artifacts: scaAgentZip
+                script {
+                    scaAgentZipRelease = "sca-agent.zip"
+                    sh "cp ${WORKSPACE}/${scaAgentZip} ${WORKSPACE}/${scaAgentZipRelease}"
+                    archiveArtifacts artifacts: scaAgentZipRelease
```
Comment (maor-rosenfeld) [inline Jenkinsfile:63]: You don't need to archive this artifact, you already archived it in the bundle stage
```suggestion
```

File: Jenkinsfile [hunk 3]
```diff
@@ -38,12 +50,18 @@ pipeline {
                 }
             }
         }
-        stage("Bundle") {
+        stage("Release") {
+            when {
+                expression {
+                    return params.Release_New_Version
```
Comment (maor-rosenfeld) [inline Jenkinsfile:56]: Groovy conventions
```suggestion
                    return params.releaseNewVersion
```

**Review by maor-rosenfeld — APPROVED** [review]

**Pull request merged**